lib/commands/location.js

```diff
@@ -0,0 +1,37 @@
+import { exec } from 'teen_process';
+import { fs, util } from 'appium-support';
+import log from '../logger';
+
+let commands = {};
+
+commands.setGeoLocation = async function (location) {
+  let {latitude, longitude} = location;
+
+  if (!util.hasValue(latitude) || !util.hasValue(longitude)) {
+    log.errorAndThrow(`Both latitude and longitude should be set`);
+  }
+
+  if (!this.isSimulator()) {
+    if (await isIdeviceLocationAvailable()) {
+      try {
+        await exec('idevicelocation', ['-u', this.opts.udid, latitude, longitude]);
```

Comment:
you could also use 

```
`${latitude}`, `${longitude}`
```

format to make sure the values were passed as strings